scripts/get_benchmark_results.py

```diff
@@ -6,12 +6,19 @@
 if __name__ == '__main__':
     output_bucket = os.getenv("OUTPUT_BUCKET")
 
+    days = os.getenv("NUMBER_OF_DAYS")
+    if days and days.isdigit() is False:
+        print("'NUMBER_OF_DAYS' environment variable must be a valid integer value")
+        sys.exit(1)
+
+    days = int(days) if days else None
```

Comment:
With the other comment in mind this might be easier
```
from_date = None
number_of_days = os.getenv("NUMBER_OF_DAYS")

    if number_of_days:
        if not isinstance(number_of_days, int):
            print("'NUMBER_OF_DAYS' environment variable must be a valid integer value")
            sys.exit(1)
        from_date = (datetime.utcnow() - timedelta(days=number_of_days))
```

Reply:
I've moved the logic, and also simplified this a bit by using the existing `parse_environment_variables` method that @MebinAbraham commented about below